Explorer/Assets/DCL/CommunicationData/URLHelpers/URN.cs

```diff
@@ -39,6 +40,9 @@ public URN(int urn)
         public override string ToString() =>
             urn;
 
+        public URN ToLower() =>
```

Comment:
This confuses me a little bit. You are creating a whole new urn struct, just using the lower case string. 
Is it possible to set the string to lower in the constructor, and just use it like that during its whole lifecycle? Therefore avoiding all the other toLower() calls in the consumers of the URN

Reply:
I followed the immutability `string` standards which also duplicates the string when lowering, that is also the most memory/cpu consuming operation. In the implementation is useful to have the duplicate since i only affect the original urn in case it exists in lower case.